plugin/tower/pkg/utils/testing/builder.go

```diff
@@ -234,6 +234,19 @@ func RuleAddPorts(rule *v1alpha1.Rule, portInfo ...string) {
 	}
 }
 
+func RuleSetDisableSymmetric(rule *v1alpha1.Rule, disableSymmetric bool) {
+	if rule == nil {
+		return
+	}
+	if len(rule.From) > 0 {
+		rule.From[0].DisableSymmetric = disableSymmetric
```

Comment:
Only first item used in Rule?

Reply:
plugin ut tool functions only support one rule.